src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1316,8 +1316,12 @@ def _validate_trusted_launch(namespace):
         return
 
     if not namespace.enable_vtpm or not namespace.enable_secure_boot:
-        logger.warning('It is recommended to specify "--enable-secure-boot True" and "--enable-secure-boot True"'
-                       ' to receive the full suite of security features that comes with Trusted Launch.')
+        logger.warning('It is recommended to specify "--enable-secure-boot True" and "--enable-vtpm True" to receive'
+                       ' the full suite of security features that comes with Trusted Launch. Please note that the'
+                       ' "--enable-secure-boot" will be enabled by default in Microsoft Ignite Event (around November)')
```

Comment:
@sirfamelin Could you help confirm that this warning log meets your expectations? If not, could you help refine it?

Reply:
LGTM